packages/govern-lib/src/GovernLib.ts

```diff
@@ -0,0 +1,136 @@
+import {
+  Address,
+  DaoData,
+  Networkish,
+  RegistryEntryData,
+  GovernQueueData,
+} from './types'
+import { ErrorInvalidNetwork, ErrorUnexpectedResult } from './errors'
+import NetworkContext from './utils/NetworkContext'
+import GraphqlClient from './GraphqlClient'
+import {
+  QUERY_DAO,
+  QUERY_DAOS,
+  QUERY_REGISTRY_ENTRY,
+  QUERY_REGISTRY_ENTRIES,
+  QUERY_QUEUE,
+  QUERY_QUEUES,
+  QUERY_QUEUES_BY_DAO,
+} from './thegraph-queries'
+
+export type GovernLibConfig = {
+  network: Networkish
+  subgraphUrl?: string
+  verbose?: boolean
+}
+
+class GovernLib {
```

Comment:
I would just call it Govern

Reply:
The idea with GovernLib is to avoid using the term Govern, so we can refer to this particular project using a specific name rather than a too generic one.

Reply:
Probably we just want to have a ``GraphQLClient`` class in the core with the ``fetchResult`` method and we use this/extend from it in the ``govern-server`` package. We could for example also change the interface of this class in the ``core`` to be aligned with the ``ClientInterface`` the ``govern.js`` package does have. The big advantage we would have with it is it would lower the code duplication in the JS side of the ``Govern`` project.